### PR TITLE
Faster exclude fields

### PR DESCRIPTION
**Pull request by Elkasitu**
Closes OSIDB-1871

### CODE REVIEW

File: osidb/filters.py [hunk 1]
```diff
@@ -170,6 +172,31 @@ def include_fields_filter(self, queryset, name, value):
         )
 
 
+class ExcludeFieldsFilterSet(SparseFieldsFilterSet):
```
Comment (jobselko) [inline osidb/filters.py:175]: This class seems to be unused (see `FlawFilter(DistinctFilterSet, IncludeFieldsFilterSet)` and similarly)

Comment (Elkasitu) [reply osidb/filters.py:175]: Good catch, thanks!

Comment (jobselko) [reply osidb/filters.py:175]: If everything that inherits from `IncludeFieldsFilterSet` will also inherit from  `ExcludeFieldsFilterSet`, could we join these two together? Or would it be a bad practice?

**Review by osoukup — APPROVED** [review]
LGTM and nice fixes on top!

**Review by jobselko — APPROVED** [review]

File: osidb/filters.py (hunk 1 above)
Comment (Elkasitu) [reply osidb/filters.py:175]: It wouldn't be bad practice, in fact we could probably invert the dependency to just have every class just use `SparseFieldsFilterSet`, but having them separate gives us the flexibility of having new views with either/or, exclude_fields is less useful than include_fields and less performant in general, so I can see a future in which new views only use `include_fields`

**Pull request merged**